src/back/routes/badges.ts

```diff
@@ -37,6 +42,28 @@ async function getBadges(req: Request<{ address: string }>): Promise<UserBadges>
   return await BadgesService.getBadges(address)
 }
 
+async function getCoreUnitsBadges() {
```

Comment:
this logic should be in the BadgesService.

Also, consider naming this `getCoreUnitBadgeSpecs`